Automattic/wp-calypso
# Adds filter to allow filtering in/out of cache templated response

This will enable easier debugging on local envs as filtering the result value to `false` will bypass the cached response.

#### Changes proposed in this Pull Request

* Add a filter around the return of the transient

#### Testing instructions

This won't be easy to test...

* Create new Page
* See Modal with some cache Template results
* Add `add_filter( 'fse_spt_vertical_templates_from_cache', '__return_false' );` to an `mu-plugin` in your WP env
* Create another new Page
* See a new non-cached response
